src/Sylius/Behat/Context/Api/AddressContext.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Context\Api;
+
+use Behat\Behat\Context\Context;
+use Sylius\Component\Core\Model\AddressInterface;
+use Sylius\Component\Core\Repository\AddressRepositoryInterface;
+
+final class AddressContext implements Context
+{
+    /** @var AddressRepositoryInterface */
+    private $addressRepository;
+
+    public function __construct(AddressRepositoryInterface $addressRepository)
+    {
+        $this->addressRepository = $addressRepository;
+    }
+
+    /**
+     * @Then /^store should contain only ("[^"]*", "[^"]*", "[^"]*", "[^"]*", "[^"]*" address)$/
+     */
+    public function storeShouldContainOnlyAddress(AddressInterface $address): void
```

Comment:
It is a domain/application test rather than api. Maybe the better way will be adding a unit test instead of behat to check the behaviour of one handler 🤔

Reply:
I thought about, I think both options are good, UnitTest need more work. We have similar context for emails that is using with api tests, I know that its mix api/domain test but is more readable that UnitTest

Reply:
The described feature definitely does not benefits a feature's actor (a Visitor). We should not use byways and focus on implementing the feature in a proper way. Therefore, as mentioned by @GSadee, this definitely should be an application/domain scenario or as a functional test.